docs/apache-airflow/howto/usage-cli.rst

```diff
@@ -374,3 +374,31 @@ JSON example output:
 
     airflow_db={"conn_type": "mysql", "login": "root", "password": "plainpassword", "host": "mysql", "schema": "airflow"}
     druid_broker_default={"conn_type": "druid", "host": "druid-broker", "port": 8082, "extra": "{\"endpoint\": \"druid/v2/sql\"}"}
+
+Testing for DAG Import Errors
+-----------------------------
+The CLI can be used to check whether any discovered DAGs have import errors via the `dags` `list-import-errors` command. The output is predictable for situations where there are no errors, such as `[]` for the json style output, so it is possible to create an automation step which fails if any DAGs cannot be imported. For instance, the check could be run in CI or pre-commit.
+
+Example command that fails if there are any errors:
+
+.. code-block:: bash
+
+    airflow dags list-import-errors | grep -q "No data found"
```

Comment:
I feel we should instead encourage using `--output=json` and tools such as `jq` for more robust parsing.

Reply:
That was actually my original approach, but i moved to this one so that i wasn't adding a new dependency. Would it make sense to add a second example using jq?

Reply:
I dislike a `grep` example too since it would produce false negatives if any of the import errors contain the message *No data found*. It would be better if it checks for the string exactly, but in general working with non-structured data should not be recommended in the first place.

Reply:
Hmm, yeah that is a fair point.

Reply:
Removed the non-jq approaches.